Sources/ProjectDescription/Template/AttributeHelpers.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+/// Error that occurs when parsing attributes from input
+enum ParsingError: Error, CustomStringConvertible, Equatable {
```

Comment:
We don't throw errors from `ProjectDescription` so we can remove this one and update all the usages accordingly.